$NetBSD: patch-ac,v 1.2 2013/11/24 05:53:52 dholland Exp $

- use (semi-)standard headers
- fix os conditionals (apparently with respect to VMS)

--- data.c.orig	Wed Feb 24 05:04:50 1999
+++ data.c
@@ -28,6 +28,7 @@
 #include "ptyx.h"		/* gets Xt stuff, too */
 #include "data.h"
 #include <setjmp.h>
+#include <sys/types.h>
 
 int bcnt = 0;
 #if !defined(NO_PATCH_CURSOR_MOVE) /* by hanmaum 1998.2.6 */
@@ -61,7 +62,7 @@
 int am_slave = 0;	/* set to 1 if running as a slave process */
 int max_plus1;
 int pty_mask;
-#if defined(linux)||defined(SVR4)||(defined(hpux)&&OSMAJORVERSION>=10)
+#ifndef VMS
 fd_set Select_mask;
 #else
 int Select_mask;
